packages/shared/src/components/checklist/NotificationChecklistStep.tsx

```diff
@@ -1,15 +1,23 @@
-import React, { ReactElement } from 'react';
+import React, { ReactElement, useContext } from 'react';
 import { ChecklistStepProps } from '../../lib/checklist';
 import { Button } from '../buttons/Button';
 import { ChecklistStep } from './ChecklistStep';
 import BellIcon from '../icons/Bell';
+import NotificationsContext from '../../contexts/NotificationsContext';
+import { NotificationPromptSource } from '../../lib/analytics';
 
 const NotificationChecklistStep = (props: ChecklistStepProps): ReactElement => {
-  // TODO WT-1293-checklist-components add business logic
+  const { onTogglePermission } = useContext(NotificationsContext);
 
   return (
     <ChecklistStep {...props}>
-      <Button icon={<BellIcon />} className="btn-primary">
+      <Button
+        icon={<BellIcon />}
+        className="btn-primary"
+        onClick={() => {
+          onTogglePermission(NotificationPromptSource.SquadPage);
```

Comment:
Would just mind checking with Sab if we wants new tracking or this is ok?
Not sure they might want a new context from checklist.